platform/qt/src/sqlite3.cpp

```diff
@@ -60,20 +60,23 @@ void checkDatabaseOpenError(const QSqlDatabase &db) {
     }
 }
 
+namespace {
+    thread_local uint64_t count = 0;
+}
+
 class DatabaseImpl {
 public:
-    DatabaseImpl(const char* filename, int flags) {
-        static uint64_t count = 0;
```

Comment:
Another way of implementing this without thread local would be using `QAtomicInt`. I'm fine with either way.